marlowe-playground-client/README.md

```diff
@@ -2,25 +2,24 @@
 
 ## Getting started
 
+**Starting the backend server**
```

Comment:
Should we add a comment that says that all commands needs to be executed inside `nix-shell`?